openupgrade_scripts/scripts/hr_attendance/17.0.2.0/post-migration.py

```diff
@@ -56,25 +56,35 @@ def fill_hr_attendance_overtime_hours(env):
 
 
 def hr_attendance_menus(env):
-    group_hr_attendance = env.ref("hr_attendance.group_hr_attendance")
-    group_hr_attendance_kiosk = env.ref("hr_attendance.group_hr_attendance_kiosk")
-    # Remove the groups of 16.0 from the Attendances menu
-    env.ref("hr_attendance.menu_hr_attendance_root").write(
-        {
-            "groups_id": [
-                Command.unlink(group_hr_attendance.id),
-                Command.unlink(group_hr_attendance_kiosk.id),
-            ]
-        }
-    )
-    # Remove the groups of 16.0 from the Attendances > Kiosk Mode menu
-    env.ref("hr_attendance.menu_hr_attendance_kiosk_no_user_mode").write(
-        {
-            "groups_id": [
-                Command.unlink(group_hr_attendance_kiosk.id),
-            ]
-        }
+    group_hr_attendance = env.ref("hr_attendance.group_hr_attendance", False)
```

Comment:
This doesn't exist nor group_hr_attendance_kiosk, because you merged into `group_hr_attendance_own_reader`.

I think you should avoid doing changes in this method. The way to proceed should be doing _xmlid_merge of the two xmlids at the last step of post-migration instead of doing in pre-migration.

Reply:
Thanks.